### DISCUSSION TITLE
Unix Binary Release (Redhat / CentOS 7.1 x86_64 RPM) does not have jpeg in delegate

### DISCUSSION THREAD

**Discussion by hourugrass** [ImageMagick/ImageMagick#2774] [Help] [answered]
This site InsiteImageMagick-7.0.10-33.x86_64.rpm, ImageMagick-libs-7.0.10-33.x86_64.rpm Download and install.
https://imagemagick.org/script/download.php

But there was no jpeg in Delegates (built-in). Isn't this rpm compatible with jpeg?

Thank you.

```
$ magick --version
Version: ImageMagick 7.0.10-33 Q16 x86_64 2020-10-04 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(4.5)
Delegates (built-in): bzlib djvu freetype ltdl png zlib
```

**Comment by hourugrass** [comment]
hello.

When I generated the rpm with the latest SRPM file, jpeg was present in the delegate.
https://imagemagick.org/download/linux/SRPMS/releases/ImageMagick-7.0.10-35.src.rpm

```
magick --version
Version: ImageMagick 7.0.10-35 Q16 x86_64 2020-11-02 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(4.5)
Delegates (built-in): bzlib cairo djvu fftw fontconfig freetype gslib jbig jng jp2 jpeg lcms ltdl lzma openexr pangocairo png ps rsvg tiff webp wmf x xml zlib
```

The official rpm file may have failed to build partly and may not be compatible with jpeg.

**Comment by urban-warrior** [comment, marked as answer]
Today's release of ImageMagick will include support for JPEG in the RPM's.  Thanks for alerting us to the problem.
